src/typedefs.php

```diff
@@ -37,6 +37,7 @@
   ?'fbonly messages' => vec<string>,
   ?'min-versions' => dict<RequirableProduct, string>,
   ?'experimental' => bool,
+  ?'default_namespace' => string,
```

Comment:
thanks just a nit: please replace underscore with space for consistency with `fbonly messages`; I see that I messed up and `min-versions` is already inconsistent, but let's not add a third way

Reply:
inherited from when this actually was yaml :p I'd also be happy for all of these to be changed to something else, I don't have any strong preference as long as they're consistent :)

Reply:
Agreed.  I just renamed it to `namespace` to avoid this :)